src/i18n/en/admin.js

```diff
@@ -588,20 +587,93 @@ export default {
     notSignedIn: 'You\'re not signed in',
     noAccess: 'Not allowed to access Crust Admin',
   },
-  'auth-settings': {
-    'external-providers': {
-      title: 'External providers',
-      enabled: 'Enable external authentication',
-      handle: 'Handle',
-      'client-key': 'Client key',
-      secret: 'Secret',
-      issuer: 'OIDC Issuer URL',
-      'provider-enabled': 'Enable this external authentication provider',
-      oidc: 'OpenID Connect',
-      gplus: 'Google',
-      facebook: 'Facebook',
-      github: 'GitHub',
-      linkedin: 'LinkedIn',
+  settings: {
+    auth: {
+      title: 'Authentication',
+      'external-providers': {
+        title: 'External providers',
+        enabled: 'Enable external authentication',
+        handle: 'Handle',
+        'client-key': 'Client key',
+        secret: 'Secret',
+        issuer: 'OIDC Issuer URL',
+        'provider-enabled': 'Enable this external authentication provider',
+        oidc: 'OpenID Connect',
+        gplus: 'Google',
+        facebook: 'Facebook',
+        github: 'GitHub',
+        linkedin: 'LinkedIn',
+      },
+      internal: {
+        title: 'Internal',
+        enabled: 'Internal authentication enabled',
+        'password-reset': {
+          enabled: 'Password reset enabled',
+        },
+        signup: {
+          enabled: 'Signup enabled',
+        },
+        'signup-email-confirmation-required': 'Signup email confirmation required',
+      },
+      frontend: {
+        title: 'Frontend',
+        url: {
+          base: 'Base',
+          'email-confirmation': 'Email confirmation',
+          'password-reset': 'Password reset',
+          redirect: 'Redirect',
+        },
+      },
+      mail: {
+        title: 'Mail',
+        'from-address': 'Sender address',
+        'from-name': 'Sender name',
+      },
+    },
+    compose: {
+      title: 'Compose',
+      panel: {
+        title: 'Panel',
+        'namespace-switcher': 'Enable namespace switcher',
+      },
+      file: {
+        title: 'File',
+        'max-size': 'Max size (MB)',
+        'type-whitelist': 'File type whitelist',
+      },
+    },
+    messaging: {
+      title: 'Messaging',
+      emoji: {
+        title: 'Emoji',
+        enabled: 'Enable emoji',
+      },
+      attachment: {
+        title: 'Attachment',
+        enabled: 'Enable attachments',
+        'max-size': 'Max size (MB)',
+        source: {
+          gallery: {
+            enabled: 'Enable attchment upload from gallery',
+          },
+          camera: {
+            enabled: 'Enable attachment upload from camera',
+          },
+        },
+        type: {
+          whitelist: 'File type whitelist',
+        },
+      },
+      notification: {
```

Comment:
browser notifications?

Reply:
Yes, do you want me to change the key name to 'browser-notifications' ?